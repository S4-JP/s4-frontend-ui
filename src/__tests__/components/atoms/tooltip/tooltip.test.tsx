import React from 'react'
import { render } from '@testing-library/react'
import { Tooltip } from '../../../../index'

describe('Tooltipのレンダリング確認', () => {
  test('propsに設定した値でレンダリングされる。', () => {
    const { getByTestId } = render(<Tooltip content="test" />)

    expect(getByTestId('atoms-tt-wrap')).toMatchSnapshot()
  })
})
