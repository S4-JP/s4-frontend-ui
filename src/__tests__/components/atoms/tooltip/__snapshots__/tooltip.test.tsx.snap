// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltipのレンダリング確認 propsに設定した値でレンダリングされる。 1`] = `
<span
  class="sc-jTYCaT oXIUP"
  data-testid="atoms-tt-wrap"
>
  <span
    class="sc-cZwWEu dRWbhx"
  >
    test
  </span>
</span>
`;
